Floofbot/Modules/Utilities.cs

```diff
@@ -99,5 +104,41 @@ public async Task RepeatMessage([Remainder] string message =null)
                 await Context.Channel.SendMessageAsync("Usage: `.say [message]`");
             }
         }
+
+        [Command("serverinfo")]
+        [Summary("Returns information about the current server")]
+        public async Task ServerInfo()
+        {
+            SocketGuild guild = Context.Guild;
+            int numberTextChannels = guild.TextChannels.Count;
+            int numberVoiceChannels = guild.VoiceChannels.Count;
+            string createdAt = $"Created {guild.CreatedAt.DateTime.ToShortDateString()}. " +
+                               $"That's over {Context.Message.CreatedAt.Subtract(guild.CreatedAt).Days} days ago!";
```

Comment:
Be careful here. We'll probably need another case for if it ends up being 1 _day_ ago